code/game/objects/structures/lspace.dm

```diff
@@ -0,0 +1,14 @@
+
+/obj/structure/lspacewall
+	name = "stack of books"
+	desc = "Large pile of books."
+	icon_state = "lspacewall"
```

Comment:
Is this a valid icon_state on its own? It might cause issues in mapping otherwise.